frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx

```diff
@@ -97,6 +98,26 @@ export const OperatorHubSubscribeForm: React.FC<OperatorHubSubscribeFormProps> =
     currentCSVDesc.annotations?.['operatorframework.io/suggested-namespace'];
   const operatorRequestsMonitoring =
     currentCSVDesc.annotations?.['operatorframework.io/cluster-monitoring'] === 'true';
+  const initializationResourceJSON =
+    currentCSVDesc.annotations?.['operatorframework.io/initialization-resource'];
+
+  let initializationResourceReference = null;
+  if (initializationResourceJSON) {
+    let initializationResource = null;
+    try {
+      initializationResource = JSON.parse(initializationResourceJSON);
+    } catch (err) {
+      // eslint-disable-next-line no-console
+      console.error(
+        err.message || 'Operator Hub Subscribe: Could not get initialization resource.',
+      );
+    }
+
+    if (initializationResource) {
+      initializationResourceReference = referenceFor(initializationResource);
+    }
```

Comment:
We can make this a `const`

```ts
const initializationResourceReference = initializationResource ? referenceFor(initializationResource) : null;
```

Reply:
This definition is inside the conditional check `if (initializationResourceJSON)` so I still need to define the value outside of that block for the code later on.  I went ahead and made this ternary though.